pkg/codegen/docs/templates/resource.tmpl

```diff
@@ -296,14 +296,20 @@ The following state arguments are supported:
 
 ## Supporting Types
 
-{{ range .NestedTypes }}
+{{ if ge (len .NestedTypes) 200 }}
+> **Note:** There are over 200 nested types for this resource.
+Only the first 200 types are included in this documentation.
```

Comment:
is it possible to use the hugo shortcode for our note styles here?

also, can we tell them what we recommend that they do to access the additional types?